src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -236,7 +237,7 @@ class CompanyStep extends React.Component {
                             this.clearErrors(_.keys(renamedValues));
                         }}
                     />
-                    <TextInput
+                    <PhoneTextInput
```

Comment:
Should we also correct the validations for this component? There won't be any special chars.